CMSgov/mint-app
# Discussion reply email to discussion originator

# EASI-3601

## Changes and Description

- New Email templates
   - new email added to email script
- New DB query to minimize db calls for reply emails



## How to test this change
1. Run the app
2. Make a discussion
3. Make a reply
   a. confirm the account you used to make the discussion is sent an email
4. Keep making replies
   a.  ensure that it always goes to the account that made the discussion
   b. ensure that the two most recent replies are displayed
   c. ensure that there is a count of all the remaining discussions.
   
The email can be validated by running scripts/dev email


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
